sda-download/internal/config/config.go

```diff
@@ -63,6 +63,8 @@ type AppConfig struct {
 	// Selected middleware for authentication and authorizaton
 	// Optional. Default value is "default" for TokenMiddleware
 	Middleware string
+
+	AllowUnencryptedDownload bool
```

Comment:
This should have better name so we don't have to change it in the near future.
Something like `ServerUnencryptedData`, then it will make sense even when we get to the point where the app only can serve encrypted *or* unencrypted data.

Reply:
Should I use the name `ServeUnencryptedData` instead of `ServerUnencryptedData`?

Reply:
Yes, the `r` is a typo, of course it should say `ServeUnencryptedData`

Reply:
addressed in the commit 749fb894